apps/web/pages/[user]/[type].tsx

```diff
@@ -358,9 +359,9 @@ async function getDynamicGroupPageProps(context: GetServerSidePropsContext) {
 }
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
-  const { user: userParam } = paramsSchema.parse(context.query);
+  const { user: userParam } = paramsSchema.parse(context.params);
```

Comment:
Made it consistent with new-booker code. Though I don't think it matters(but didn't want to test again as it takes time to build on vercel and then test.)